### DISCUSSION TITLE
saf-training/courses/user/02

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#40] [Ideas]
# saf-training/courses/user/02

The MITRE Security Automation Framework Team's Introduction to InSpec profile development

https://mitre.github.io/saf-training/courses/user/02.html

**Comment by karikarshivani** [comment]
For 2.2, should 1b and 2a be swapped?

**Comment by karikarshivani** [comment]
There is a more updated graphic for the pillars (with OHDF) that can be used
